edit-post/store/effects.js

```diff
@@ -1,7 +1,8 @@
 /**
  * External dependencies
  */
-import { reduce, values, some } from 'lodash';
+import { reduce, some } from 'lodash';
+import 'formdata-polyfill';
```

Comment:
Could we polyfill this as we do other features already in `client-assets.php` via `gutenberg_get_script_polyfill` for (a) consistency and (b) not loading / including the resource for browsers which _do_ support the feature?

Reply:
Are you certain this technique (`gutenberg_get_script_polyfill`) is working properly, I just tried it for formdata but it didn't work. Also I'm wondering if it works for the other polyfills since I don't see any request for the said scripts (promises and fetch).

We do use another polyfill using the import technique `element-closest`. I'm thinking we should consolidate these separately, and my idea, for now, is to register a `wp-polyfills` script depending on all polyfills and add it as a dependency to all our other scripts.

Reply:
I think the issue is that `FormData` is there, but has a very limited implementation. See: https://developer.mozilla.org/en-US/docs/Web/API/FormData#Browser_compatibility. I guess it would need to have a more complex detection strategy,

Reply:
@gziolo I don't think so because I've use `true` to always load it.

Reply:
Stupid me :) I'll try again